apis/extension/resource.go

```diff
@@ -33,6 +33,9 @@ const (
 	BatchCPU    corev1.ResourceName = ResourceDomainPrefix + "batch-cpu"
 	BatchMemory corev1.ResourceName = ResourceDomainPrefix + "batch-memory"
 
+	NvidiaGPU corev1.ResourceName = "nvidia.com/gpu"
+	KoordGPU  corev1.ResourceName = ResourceDomainPrefix + "gpu"
```

Comment:
the prefix is `kubernetes.io/gpu`, but the following resource name will be `kubernetes.io/gpugpucore`...

Reply:
sorry, i made a mistake

Reply:
```
	KoordGPU       = "kubernetes.io/gpu"
	GPUCore        = "kubernetes.io/gpu-core"
	GPUMemory      = "kubernetes.io/gpu-memory"
	GPUMemoryRatio = "kubernetes.io/gpu-memory-ratio
```

Reply:
> sorry, i made a mistake

it's ok, I forgive you. we're family. 😉